content/graphql/field-middleware.md

```diff
@@ -50,7 +50,7 @@ const value = await next();
 return value?.toUpperCase();
 ```
 
-In this case, every title will be automatically uppercased, when requested.
+In this case, every title will be automatically be uppercased, when requested.
```

Comment:
will **be** automatically **be** uppercased? 🤔

Reply:
Oops, sorry about that, I'll fix it when I get a chance.